crates/solidity/testing/sourcify/src/sourcify.rs

```diff
@@ -59,15 +53,42 @@ impl Manifest {
         })
     }
 
+    fn obtain_manifest(options: &ArchiveOptions) -> Result<serde_json::Value> {
+        let manifest_path = CargoWorkspace::locate_source_crate("solidity_testing_sourcify")
+            .unwrap_or_default()
+            .join("target/manifest.json");
+
+        if !options.offline && !file_exists_and_is_fresh(&manifest_path) {
```

Comment:
> `--offline` will not download any new files and fail if the required files are not already present

IIUC, this silently skips it if they don't exist?

Reply:
In this case it will fail a couple of lines below, when trying to parse the manifest, but you have a point. I'll improve this and the handling of the shard archives.